src/otx/core/model/module/action_classification.py

```diff
@@ -30,32 +32,40 @@ def __init__(
         torch_compile: bool,
         optimizer: list[OptimizerCallable] | OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
         scheduler: list[LRSchedulerCallable] | LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
+        metric: Metric = Accuracy,
```

Comment:
Define `MetricCallable` type such as `LRSchedulerCallable` or `OptimizerCallable`
```suggestion
        metric: list[MetricCallable] | MetricCallable = Accuracy,
```

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/2917/commits/da532871951921f22dc38a1c9b49f1f7e5130deb. Supporting the list of metric will be followed.